### PR TITLE
added logo with text

### PR DESCRIPTION
**Pull request by nivyapeter**

### CODE REVIEW

File: azure-page/src/App.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import logo from "./logo.svg";
+import img from "./manage.png";
+import "./index.css";
+// import Dropdown from "./components/dropdown/explore/explore";
+import Header from "./components/header";
+import Banner from "./components/banner";
+import Card from "./components/Card";
+// import Explore from "./components/dropdown/explore/explore";
+// import DropDown from "./components/dropdown/explore/DropdownMain";
```
Comment (vonnue-sijin) [inline azure-page/src/App.js]: No commented codes for review. Remove all, o/w mention why its kept commented.

File: azure-page/src/App.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+const title = "Azure";
+const link1 = "https://microsoft.com/";
+const link2 = "https://azure.microsoft.com/en-gb/";
```
Comment (vonnue-sijin) [inline azure-page/src/App.js]: No magic string, move all to a constants file

File: azure-page/src/App.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+
+function App() {
+  return (
+    <div className="App container">
```
Comment (vonnue-sijin) [inline azure-page/src/App.js]: Follow CSS convention for classname naming.
Should be "app"

all lower case separated by dashes. 
 Follow
kebab-case. Also called caterpillar-case, dash-case, hyphen-case, lisp-case, spinal-case and css-case

File: azure-page/src/components/header.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+
+
+    </div><div className="dropdown-section">
+        {/* <Explore names={names}/> */}
+        {componentRender}
```
Comment (vonnue-sijin) [inline azure-page/src/components/header.js:106]: Wont a ternary operator do the job here?

File: azure-page/src/index.css [hunk 5]
```diff
@@ -0,0 +1,248 @@
+body {
```
Comment (vonnue-sijin) [inline azure-page/src/index.css:1]: Not all styles goes to index.css

**Review by vonnue-sijin — COMMENTED** [review]
The issues I commented are repeating across the PR please fix all of them. Also format the code before sending for review

File: azure-page/src/App.js (hunk 1 above)
Comment (nivyapeter) [reply azure-page/src/App.js]: ok.Resolved

File: azure-page/src/App.js (hunk 3 above)
Comment (nivyapeter) [reply azure-page/src/App.js]: Resolved

**Pull request merged**